code/modules/mob/living/carbon/human/species_types/ethari.dm

```diff
@@ -35,4 +35,12 @@
 
 /datum/species/ethari/spec_death(gibbed, mob/living/carbon/human/H)
 	if(H)
-		H.endTailWag()
\ No newline at end of file
+		H.endTailWag()
+
+/datum/species/ethari/handle_chemicals(datum/reagent/chem, mob/living/carbon/human/H)
+	if(chem.id == "cocoa" || chem.id == "hot_coco")
```

Comment:
This feels like the wrong way to do this, as all other chems etc are handled elsewhere.... either that, or you're overriding chem handling without also including parents...

Reply:
This is the only proper way to handle species-based reagents handling.

The only thing that's overwritten here is exotic blood handling which doesn't apply to Ethari anyway.

![Here](https://github.com/OracleStation/OracleStation/blob/6345060bfb1c596081ef0e3d0bc1c6bd00df20aa/code/modules/mob/living/carbon/human/species_types/podpeople.dm#L45 is an example of how it's used.

We could possibly implement a more complicated allergy system down the road, but I don't think it's our main focus for now.